request/graphql/schema/collection.go

```diff
@@ -24,9 +24,32 @@ import (
 	"github.com/graphql-go/graphql/language/source"
 )
 
+type collectionDefinition struct {
```

Comment:
thought: You'll be able to use the one in the client package after you move it there instead of redefining it here.